src/pages/PushNofiPage/index.tsx

```
@@ -0,0 +1,10 @@
+import { SafeAreaView } from 'react-native-safe-area-context';
+import useGetPatientList from '../PatientListPage/hooks/useGetPatientList';
+import { styles } from '../PatientListPage/styles';
+
+const list = () => {
+  const result = useGetPatientList();
```

ESLint

[error] 'result' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)